templates/index.tpl

```diff
@@ -25,8 +25,8 @@
                         {t}Audio only (MP3){/t}
                     </label>
                     <div class="seekOptions">
-                        {t}From{/t} <input type="text" placeholder="00:00:00" value="" name="from" />
```

Comment:
Why did you remove the `/` add the end of the tag?

Reply:
🤦‍♂️